dgc-testdata-tests/DGCTestDataVerifier.cs

```diff
@@ -0,0 +1,81 @@
+﻿using DCC;
+using DGC;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace dgc.testdata.tests
+{
+    public class DGCTestDataVerifier
+    {
```

Comment:
Similarly namespace + class names which should be just some ReSharper re-factor work AFAICS